### DISCUSSION TITLE
It even runs on PowerPC Linux. Thanks!

### DISCUSSION THREAD

**Discussion by wicknix** [uowuo/abaddon#190] [Show and tell]
I've been able to build Abaddon on both Debian Sid and Void Linux (PowerPC ports). I edited the main.css a bit though to make the colors closer to the official app. Thanks for this!

Here it is running on my ancient Macintosh PowerBook G4 on Debian.

![abaddon-powerpc](https://github.com/uowuo/abaddon/assets/39230578/edf8108e-c7d7-4a66-aa5c-011ba2fde501)

**Comment by ouwou** [comment]
nice, looks quite good!
